### PR TITLE
BaSP-MR-09-32: Member post

### PR DESCRIPTION
**Pull request by EmaBaldor**
Added routes controllers and validations for post

### CODE REVIEW

**Review by JuanMornik — APPROVED** [review]

**Review by NicoCagnina — APPROVED** [review]

**Review by JuanMornik — CHANGES_REQUESTED** [review]
The code in the PR name is wrong. The correct code is 32

File: src/validations/members.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        'number.max': 'Phone number must be at most 10 digits',
+      }),
+    email: Joi.string().email().required(),
+    city: Joi.string().min(3).max(15).required(),
+    postal_code: Joi.string().min(4).max(5).required()
+      .messages({
+        'number.min': 'Postal code must be at least 4 digits',
+        'number.max': 'Postal code must be at most 5 digits',
+      }),
+    is_active: Joi.boolean().required(),
```
Comment (NicoCagnina) [inline src/validations/members.js:20]: remember to use camelCase in complex words.

File: src/validations/members.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    lastName: Joi.string().min(3).max(15).required(),
+    birthday: Joi.date().required(),
+    phone: Joi.string().length(10).required()
+      .messages({
+        'number.min': 'Phone number must be at least 10 digits',
+        'number.max': 'Phone number must be at most 10 digits',
+      }),
+    email: Joi.string().email().required(),
+    city: Joi.string().min(3).max(15).required(),
+    postal_code: Joi.string().min(4).max(5).required()
```
Comment (NicoCagnina) [inline src/validations/members.js:15]: remember to use camelCase in complex words.

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/routes/trainer.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 const express = require('express');
-const validateUpdate = require('../validations/trainer');
+// const validateUpdate = require('../validations/trainer');
```
Comment (NicoCagnina) [inline src/routes/trainer.js:2]: remove this comment

File: src/routes/index.js [hunk 4]
```diff
@@ -22,5 +23,7 @@ router.use('/activity', activities);
 router.use('/admins', admins);
 
 router.use('/class', classes);
-router.use('/trainer', trainerRoutes);
+
+// router.use('/trainer', trainerRoutes);
```
Comment (NicoCagnina) [inline src/routes/index.js:27]: remove this comment

File: src/routes/index.js [hunk 5]
```diff
@@ -1,5 +1,6 @@
 const express = require('express');
-const trainerRoutes = require('./trainer');
+
+// const trainerRoutes = require('./trainer');
```
Comment (NicoCagnina) [inline src/routes/index.js:3]: remove this comment

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/controllers/member.js [hunk 6]
```diff
@@ -1,5 +1,42 @@
 const Member = require('../models/Member');
 
+const createMember = (req, res) => {
+  const {
+    firstName,
```
Comment (NicoCagnina) [inline src/controllers/member.js:5]: this name should use camelCase

File: src/controllers/member.js [hunk 7]
```diff
@@ -1,5 +1,42 @@
...
+    lastName,
+    dni,
+    birthday,
+    phone,
+    email,
+    city,
+    postalCode,
```
Comment (NicoCagnina) [inline src/controllers/member.js:12]: this should be status code 404

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

**Review by NicoCagnina — APPROVED** [review]
Looks good to me!

**Review by JuanMornik — APPROVED** [review]

**Pull request merged**